zeppelin-web/add_dependencies.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+( sudo apt-get install libfontconfig -y || sudo yum install fontconfig -y ) || exit 0;
```

Comment:
Then how about print message "Please install xxxxx with apt-get install xxxxx command" when required libraries are not available on the system?

I still think installing package without asking user is not really a good idea. And we can't assume that user will build Zeppelin with 'root' account.